src/pages/home/report/ReportActionContextMenu.js

```diff
@@ -148,10 +158,29 @@ class ReportActionContextMenu extends React.Component {
         return lodashGet(message, 'text', '');
     }
 
+    /**
+     * Can the current user edit this report action?
+     *
+     * @return {Boolean}
+     */
+    canEdit() {
+        return this.props.reportAction.actorEmail === this.props.session.email
```

Comment:
@sketchydroide we also shouldn't be able to edit/delete IOU actions, or any other type of report action (not yet, anyways). So let's add a check for that here:

```javascript
canEditReportAction() {
    return this.props.reportAction.actionName === 'ADDCOMMENT' // Bonus: make a constant for report action names
        && this.props.reportAction.actorEmail === this.props.session.email
        && this.props.reportAction.reportActionID;
}
```